modules/core/server/views/email-templates/signup-reminder.server.view.html

```diff
@@ -1,127 +1,27 @@
-{% extends './email.server.view.html' %}
+{% extends './email-with-container.server.view.html' %}
 
-{% block content %}
+{% block textcontent %}
+  <p>{{name}},</p>
 
-  <!-- MODULE ROW // -->
-  <tr>
-    <td align="center" valign="top">
-        <!-- CENTERING TABLE // -->
-          <!--
-            The centering table keeps the content
-              tables centered in the emailBody table,
-              in case its width is set to 100%.
-          -->
-        <table border="0" cellpadding="0" cellspacing="0" width="100%">
-            <tr>
-                <td align="center" valign="top">
-                    <!-- FLEXIBLE CONTAINER // -->
-                      <!--
-                        The flexible container has a set width
-                          that gets overridden by the media query.
-                          Most content tables within can then be
-                          given 100% widths.
-                      -->
-                    <table border="0" cellpadding="0" cellspacing="0" width="600" class="flexibleContainer">
-                        <tr>
-                            <td align="center" valign="top" width="600" class="flexibleContainerCell">
+  <p>We haven't heard from you since you signed up to Trustroots {{timeAgo}}.</p>
 
+  <p>Your profile will not be visible to others if you don't confirm your email address ({{email}}).</p>
 
-                                  <!-- CONTENT TABLE // -->
-                                  <!--
-                                    The content table is the first element
-                                      that's entirely separate from the structural
-                                      framework of the email.
-                                  -->
-                                  <table border="0" cellpadding="0" cellspacing="0" width="100%">
-                                      <tr>
-                                          <td valign="top" class="textContent">
+  <p>It's easy — just click on the link below.</p>
 
-                                              <p>{{name}},</p>
-                                              <p>We haven't heard from you since you signed up to Trustroots {{timeAgo}}.</p>
-                                              <p>Your profile will not be visible to others if you don't confirm your email address ({{email}}).</p>
-                                              <p>It's easy — just click on the link below.</p>
-                                          </td>
-                                      </tr>
-                                  </table>
-                                  <!-- // CONTENT TABLE -->
-
-
-                              </td>
-                          </tr>
-                      </table>
-                      <!-- // FLEXIBLE CONTAINER -->
-                  </td>
-              </tr>
-          </table>
-          <!-- // CENTERING TABLE -->
-      </td>
-  </tr>
-  <!-- // MODULE ROW -->
+  <p><small><em>
```

Comment:
We're changing the order of the message here.

* the first part
* action button
* the second part

gets changed to
* first part
* second part
* action button

It seems that we'll need to change the `email-with-container` to include an optional block e.g. `contentafterbutton`, which could include the part after button.

Reply:
Yup, as noted in the commit. I preferred to refactor it, and just chose this way. But can do another block to not reorder!

Reply:
Yeaaah I think for the sake of simplicity this change is okay :thumbsup:

Probably good anyway to have call-to-action button always the last thing after any content.